app/forms/support_interface/application_forms/pick_course_form.rb

```diff
@@ -10,7 +10,7 @@ class PickCourseForm
       validates :course_code, presence: true
       validate :course_is_open_on_apply, on: :save
 
-      RadioOption = Struct.new(:course_option_id, :course_name, :course_code, :site_name)
+      RadioOption = Struct.new(:course_option_id, :provider_name, :provider_code, :course_name, :course_code, :site_name, :study_mode)
```

Comment:
regular args get a bit error prone when you have this many. Not a big problem here because this struct is only used in it's parent class. Alternative (had to look this up) is to use `keyword_init: true` to switch the struct initializer over to keyword args.

Reply:
Yea good shout. Have sorted it